_tools/release/01_bump_version.ts

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-run=git --no-check
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { getReleasesMdFile, loadRepo, VersionFile } from "./repo.ts";
+
+const repo = await loadRepo();
+const versionFile = new VersionFile();
+const currentVersion = versionFile.version;
+const newVersion = versionFile.version.inc(getVersionIncrementKind());
```

Comment:
Just to ensure, this is being incremented without using `v` prefix?

Reply:
Yup and this will tag without a `v` prefix.